src/main/resources/application.yml

```diff
@@ -18,6 +18,30 @@ spring:
       test:
         - "stdout"
 
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          jwk-set-uri: ${oauth.endpoint.url}/.well-known/jwks.json
+
+      client:
+        registration:
+          prison-api:
+            provider: hmpps-auth
+            client-id: ${hmpps-auth.client.client-id}
+            client-secret: ${hmpps-auth.client.client-secret}
+            authorization-grant-type: client_credentials
+            scope: read,write
+          court-register-api:
+            provider: hmpps-auth
+            client-id: ${hmpps-auth.client.client-id}
+            client-secret: ${hmpps-auth.client.client-secret}
+            authorization-grant-type: client_credentials
+            scope: read
```

Comment:
might be best to remove scope and allow Auth to decide scope? We have had issues of clients accidentally requesting a "lower" scope causing obscure bugs

Reply:
ok thanks